src/main/docker/Dockerfile

```diff
@@ -22,8 +22,11 @@ MAINTAINER Cyface GmbH <mail@cyface.de>
 # Create user. Unproblematic default id (wikipedia.org/wiki/User_identifier)
 ARG USER_ID=9999
 ARG GROUP_ID=9999
-RUN addgroup --gid $GROUP_ID user
-RUN adduser --disabled-password --gecos '' --uid $USER_ID --gid $GROUP_ID user
+# For some reasons `addgroup` and `adduser` are not available anymore in the base image.
+RUN groupadd --gid $GROUP_ID user
+RUN useradd -c '' --uid $USER_ID --gid $GROUP_ID --no-create-home --shell /usr/sbin/nologin user
+# The `useradd` version does not support `--disabled-password`, so we lock the user password instead:
+RUN passwd -l user
```

Comment:
I tried to install `addgroup` and `adduser` (package `password`) but it's already installed.
I also tried to declare the full path of those programs, but it also did not work.
I guess the lastest base image did break something here.

So I switched to `groupadd` and `useradd` equivalents which seem to work